app/assets/stylesheets/partials/_click_to_play.scss

```diff
@@ -10,6 +10,8 @@
     .img-container {
       img {
         width : 100%;
+        height: 100%;
+        object-fit: contain;
```

Comment:
I knew it existed but didn't know its name or how to use it until yesterday. Yeah in IE things are going to be stretched but normally it should be minor since people ought to be matching the aspect ratio of the image to the interactive.